src/main/java/org/folio/consortia/service/impl/UserAffiliationAsyncServiceImpl.java

```diff
@@ -38,7 +38,7 @@ public CompletableFuture<Void> createPrimaryUserAffiliationsAsync(UUID consortiu
     Tenant tenantDto) {
     return CompletableFuture.runAsync(() -> {
         log.info("Start creating user primary affiliation for tenant {}", tenantDto.getId());
-        var users = userService.getUsersByQuery(StringUtils.EMPTY, 0, Integer.MAX_VALUE);
+        var users = userService.getUsersByQuery("cql.allRecords=1", 0, Integer.MAX_VALUE);
```

Comment:
it didn't help, error is the same

Reply:
error comes from https://github.com/folio-org/folio-spring-support/blob/master/folio-spring-base/src/main/java/org/folio/spring/client/EnrichUrlAndHeadersClient.java#L39 
smth connected to context

Reply:
I think CompletableFuture don't pass folioExecutionContext
![image](https://github.com/folio-org/mod-consortia/assets/113523904/0b2e890e-bf04-45cf-8d0e-90050262b64a)